src/views/StoryViewer.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <div class="viewer" :style="currentFrame.style">
+  <div class="viewer" :style="currentFrame.style" @mousedown="handlePause" @touchstart="beginPause" @touchend="endPause">
     <div class="progress-container">
       <div v-for="n in storyLength" :key="n" class="progress">
-        <div v-if="currentFrame.id === n" class="loading"></div>
+        <div v-if="currentFrame.id === n" :class="{ loading: true, paused: isPaused}" :style="{ animationDuration: animationDuration }"></div>
```

Comment:
It looks like this <div> and the <div> below could be merged together without v-if/v-else-if by adding the currentFrame.id conditions to the `:class`.

Not 100% related to this change by a good opportunity to simplify.

Reply:
Oh I see what you're saying, yes much better, thanks